src/Raven.Client/Documents/Operations/Backups/BackupProgress.cs

```diff
@@ -11,6 +11,8 @@ public class BackupProgress : SmugglerResult.SmugglerProgress
 
         public UploadToAzure AzureBackup => ((BackupResult)_result).AzureBackup;
 
+        public UploadToGoogleCloudStorage GoogleCloudStorageBackup => ((BackupResult)_result).GoogleCloudStorageBackup;
```

Comment:
Rename:
- type to `UploadToGoogleCloud` 
- property to `GoogleCloudBackup`

Reply:
Done.